demo/x-local-translate.html

```diff
@@ -76,6 +77,10 @@
         }
       },
 
+      _changeLanguage() {
+        this.$.switch.checked = this.language === 'fr';
```

Comment:
I think this is working as expected. What I am seeing is that manually changing the toggle from the UI triggers a call to _toggle which changes this.language. That triggers the language to be changed in all instances of app-localize-behavior which triggers a call to _changeLanguage in the two demo components. _changeLanguage sets the state of the toggle to match the current language. The key seems to be that doing this does not trigger another call to _toggle, so there is no infinite recursion.